src/Interfaces.jl

```diff
@@ -158,16 +193,24 @@ function toimplements!(IT, arg::Expr, shouldthrow::Bool=true)
             arg = arg.args[2]
         end
         arg.args[2] = toimplements!(IT, arg.args[2], false)
-        return :(($origarg) || Interfaces.atleastonerequired($IT, $(Meta.quot(argcopy))))
+        return quote
+            check = $origarg
+            check || Interfaces.atleastonerequired(debuglvl, $IT, $(Meta.quot(argcopy)))
+            check
```

Comment:
ah, yes, this patten that appears everywhere:
```jl
check = ...
check || ...
check 
```
is because we need each block to evaluate to / terminate in a `Bool` (hence the trailing `check`), because of how our `||` expressions work